forem/forem
# Billboards type_of: added field to the API and small admin form changes

## What type of PR is this? (check all applicable)


## Description
- updated `type_of` radio buttons labels slightly
- removed "For internal use" from `/admin/customization/display_ads/new`
- allowed accessing and setting billboards `type_of` via api
- updated the api docs

## Related Tickets & Documents
- Closes [#534](https://github.com/forem/forem-internal-eng/issues/534)

## QA Instructions, Screenshots, Recordings
- Small changes in the `/admin/customization/display_ads/new` (and edit) forms
![изображение](https://user-images.githubusercontent.com/30115/219367218-1749de39-bc71-4368-95cd-0ad2fc784e37.png)
- check the swagger api display_ad docs

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
